### PR TITLE
fix: fixed the cursor not moving to next line after inserting divider

### PR DESCRIPTION
**Pull request by Mukund-Tandon**
fixes: #370 
Before:
ezgif com-video-to-gif (3)

After -

https://github.com/AppFlowy-IO/appflowy-editor/assets/71614009/0f0a2de1-533a-4356-8337-48323935b081

### CODE REVIEW

**Review by MayurSMahajan — CHANGES_REQUESTED** [review]
Great work! Can you add a test to assure this functionality, so that if it breaks in the future, we would know.

**Comment by Mukund-Tandon** [comment]
> Great work! Can you add a test to assure this functionality, so that if it breaks in the future, we would know.

Ok,I will add them

**Review by Xazin — APPROVED** [review]

File: lib/src/editor/block_component/divider_block_component/divider_block_component.dart [hunk 1]
```diff
@@ -96,13 +96,13 @@ class _DividerBlockComponentWidgetState
...
-        child: child,
-      );
-    }
+    // if (widget.showActions && widget.actionBuilder != null) {
+    //   child = BlockComponentActionWrapper(
+    //     node: node,
+    //     actionBuilder: widget.actionBuilder!,
+    //     child: child,
+    //   );
+    // }
```
Comment (LucasXu0) [inline lib/src/editor/block_component/divider_block_component/divider_block_component.dart:105]: Why do you comment out this code?

File: lib/src/editor/block_component/divider_block_component/divider_block_component.dart [hunk 2]
```diff
@@ -120,7 +120,7 @@ class _DividerBlockComponentWidgetState
   bool get shouldCursorBlink => false;
 
   @override
-  CursorStyle get cursorStyle => CursorStyle.cover;
+  CursorStyle get cursorStyle => CursorStyle.borderLine;
```
Comment (LucasXu0) [inline lib/src/editor/block_component/divider_block_component/divider_block_component.dart:123]: Don't modify it

File: lib/src/editor/block_component/divider_block_component/divider_block_component.dart (hunk 1 above)
Comment (Mukund-Tandon) [reply lib/src/editor/block_component/divider_block_component/divider_block_component.dart:105]: I am sorry ,forgot to uncomment it , I was just testing and understanding the code base

**Review by MayurSMahajan — APPROVED** [review]

**Pull request merged**